package.json

```diff
@@ -4,8 +4,12 @@
   "description": "Deferral poject",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node ./build/index.js",
+    "dev": "nodemon --exec babel-node server/index.js",
```

Comment:
Great job. I only have an issue with the npm script.
Would have preferred the conventional `startDev`  or  `devStart` instead of just "dev".

Reply:
Do we go with the `npm run dev` or `npm run startDev` ?